src/utility/api.test.js

```diff
@@ -0,0 +1,13 @@
+import { Get } from './api';
+import Url from './url'
+
+const NONSENSE_URL = "asldfjaslkdfj.afi/asflasljlkjlijljmnxhauserhlidf";
```

Comment:
Thoughts on moving these urls to something like a constants.js file?

Reply:
oops made this same comment in `url.js`. i'd really like to see this